e2e/cypress/e2e/cclw/pages/family.cy.js

```diff
@@ -8,7 +8,7 @@ const relatedDocsSelector = '[data-cy="related-documents"]';
 const targetsSelector = '[data-cy="targets"]';
 
 const pageSelectors = ["h1", '[data-cy="country-link"]', '[data-cy="download-target-csv"]', mainDocsSelector, relatedDocsSelector, targetsSelector];
-const pageHeadings = ["Main documents", "Related documents", "Targets", "Timeline", "Note"];
+const pageHeadings = ["Main document", "Other documents in this entry", "Targets", "Timeline", "Note"];
```

Comment:
Sure, ‘related’ was not favoured because it implied a connection with a different document rather than the same ‘document’ (or what we call family).

And semantically saying “Other documents in this document” seems off.